marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserArtifactsCommand.java

```diff
@@ -382,6 +378,25 @@ protected ObjectNode replaceLanguageWithLang(ObjectNode object) {
         return newObject;
     }
 
+    /**
+     * Reads the artifact file, replaces tokens and then returns the content as a JsonNode.
+     *
+     * @param file
+     * @return
+     */
+    private JsonNode readArtifact(File file) {
+        JsonNode jsonNode;
```

Comment:
This is fine to keep, but generally I think it's simpler to just do "return objectMapper.readTree" as opposed to declaring this variable here. Shortens the method too.